src/plugins/presentation_panel/public/panel_actions/customize_panel_action/customize_panel_editor.tsx

```diff
@@ -140,42 +139,48 @@ export const CustomizePanelEditor = ({
             />
           }
           labelAppend={
-            <EuiButtonEmpty
-              size="xs"
-              data-test-subj="resetCustomEmbeddablePanelTitleButton"
-              onClick={() => setPanelTitle(api.defaultPanelTitle?.value)}
-              disabled={hideTitle || !editMode || api?.defaultPanelTitle?.value === panelTitle}
+            api?.defaultPanelTitle?.value && (
+              <EuiButtonEmpty
+                size="xs"
+                data-test-subj="resetCustomEmbeddablePanelTitleButton"
+                onClick={() => setPanelTitle(api.defaultPanelTitle?.value)}
+                disabled={hideTitle || panelTitle === api?.defaultPanelTitle?.value}
+                aria-label={i18n.translate(
+                  'presentationPanel.action.customizePanel.flyout.optionsMenuForm.resetCustomTitleButtonAriaLabel',
+                  {
+                    defaultMessage: 'Reset title to default',
+                  }
+                )}
+              >
+                <FormattedMessage
+                  id="presentationPanel.action.customizePanel.flyout.optionsMenuForm.resetCustomTitleButtonLabel"
+                  defaultMessage="Reset to default"
+                />
+              </EuiButtonEmpty>
+            )
+          }
+        >
+          <EuiFormControlLayout
+            clear={{ onClick: hideTitle ? undefined : () => setPanelTitle('') }}
+          >
```

Comment:
The 'Clear input' button in the title text box feels unnecessary to me. But I'm not a designer. @andreadelrio what do you think?

Reply:
Yeah could be unnecessary, at first I thought reset meany clear input but that's not what it does so I just added the clear to differentiate it from the reset button. See what the designers thinks.

Reply:
I think it might generate some confusion with the `Show title` toggle. It feels a bit unnecessary to me.

Reply:
Ok sounds good, removed in 7634909963319c8c6d06653723f551f1dba12657